launcher/modplatform/flame/FlameInstanceCreationTask.cpp

```diff
@@ -354,6 +354,8 @@ bool FlameCreationTask::createInstance()
         auto id = loader.id;
         if (id.startsWith("neoforge-")) {
             id.remove("neoforge-");
+            if (id.startsWith("1.20.1-"))
+                id = id.mid(7);  // this is a mess for curseforge
```

Comment:
Can you not use remove here for consistency?

Reply:
Like this?
```suggestion
                id.remove("1.20.1-");  // this is a mess for curseforge
```

Reply:
yeah?